aptos-move/move-examples/data_structures/sources/big_vector.move

```diff
@@ -11,10 +11,10 @@ module aptos_std::big_vector {
     const EVECTOR_NOT_EMPTY: u64 = 3;
 
     /// Index of the value in the buckets.
-    struct BigVectorIndex has copy, drop, store {
-        bucket_index: u64,
-        vec_index: u64,
-    }
+//    struct BigVectorIndex has copy, drop, store {
+//        bucket_index: u64,
+//        vec_index: u64,
+//    }
```

Comment:
sup?

Reply:
hmmmm, reverted it back.. I forgot I was editing some files before :(